cloudinit/sources/DataSourceOVF.py

```diff
@@ -684,4 +740,69 @@ def _raise_error_status(prefix, error, event, config_file, conf):
     util.del_dir(os.path.dirname(config_file))
     raise error
 
+
+def load_cloudinit_data(md_path, ud_path):
+    """
+    Load the cloud-init meta data, user data, cfg and network from the
+    given files
+    """
```

Comment:
more helpful docstrings
```suggestion

    @return: 4-tuple of configuration
        metadata, userdata, cfg={}, network
    @raises: FileNotFoundError if md_path or ud_path are absent
    """
```